src/locales/en.json

```diff
@@ -172,7 +172,7 @@
   },
```

Comment:
Has this file been updated automatically by bisu?

Reply:
yes

Reply:
So which command do you run to update the it? Should we add it to it to the package.json?

Reply:
you just have to run `bisu` on the root of the project. It's a ruby gem. Is a package.json command something we want? If so how do we ensure in this project that ruby gem is installed prior to running the command ? I can't see a `Gemfile` in the project

Reply:
on mobiles we only run it explicitly when required, i.e. string update. On the iOS SDK we have a Gemfile, on Android we don't. I assume on Android its a manual installation.